homebrew.py

```diff
@@ -0,0 +1,138 @@
+from re import search
+from sys import platform
+from subprocess import check_output, CalledProcessError
+
+
+class HomeBrew:
+    def __init__(self):
+        self._libdirs = self.libdirs
+        self._incdirs = self.incdirs
+        self._check_brew_isinstalled()
```

Comment:
Don't forget to also call the other helpers here to:

`    def _check_netsnmp_isinstalled(self) -> list[str]:`

`    def _check_openssl_isinstalled(self) -> list[str]:`

Reply:
These two operations calling from related functions.